### ISSUE TITLE
Alternating template is not displaying a Text Field Correctly

### ISSUE THREAD

**Issue by andrewwpaction** [dudaster/ele-custom-skin#218]
I have set up an alternating templates post on the home page of my site reefmarine.net. It consists of 1 column and 3 posts per page. I have checked the loop template and if I test the loop by changing the Preview setting to the post that not display correctly and it works fine. 

I am stumped as to why this is happening. Just wondering if there may be an issue with the alternating template option.

Website: https://reefmarine.net

![Home Page Loop missing text](https://user-images.githubusercontent.com/63601336/82871504-06a9c200-9f75-11ea-9d52-5098f6c62ff5.jpg)
![Home Page Loop](https://user-images.githubusercontent.com/63601336/82871508-090c1c00-9f75-11ea-903e-54c8d5b0197c.jpg)

**Comment by dudaster** [comment]
You sent me a screenshot of the loop that is working. Please make sure you gave the alternating loop text widget the proper dynamic value.

**Comment by andrewwpaction** [comment]
I don’t understand,



The first image with the MISSING in it is the home page, the second image
is the loop template working in Elementor. So the text that is missing in
the first (home page) image is shown in the second image of the same post
working in Elementor.



The attached is the being edited in Elementor, the Custom Post that
supplies the data and the edit page. Why would the Title and button text
display but not the content text?



Link to page: https://reefmarine.net/



Talk soon,







E: andrew@wpaction.com.au

Mb: 0417 115 640

Web: wpaction.com.au



*From:* Liviu Duda <notifications@github.com>
*Sent:* Tuesday, 26 May 2020 5:33 PM
*To:* dudaster/ele-custom-skin <ele-custom-skin@noreply.github.com>
*Cc:* andrewwpaction <andrew@wpaction.com.au>; Author <
author@noreply.github.com>
*Subject:* Re: [dudaster/ele-custom-skin] Alternating template is not
displaying a Text Field Correctly (#218)



You sent me a screenshot of the loop that is working. Please make sure you
gave the alternating loop text widget the proper dynamic value.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/dudaster/ele-custom-skin/issues/218#issuecomment-633860033>,
or unsubscribe

**Comment by dudaster** [comment]
As I can see you're using to Loop Templates: 
- one with the image on the right 
- and the other with the image on the left.

The one with the image on the left is not working. Chek the Text Editor widget if it has the proper dynamic value set there.

**Comment by andrewwpaction** [comment]
Yes it does see attached. Text is coming for the Post Content Field. All
three Posts do or it would not work. That is what is confusing. It’s a loop
of the three post they all use the same fields.



Am I missing something?



Talk soon,







E: andrew@wpaction.com.au

Mb: 0417 115 640

Web: wpaction.com.au



*From:* Liviu Duda <notifications@github.com>
*Sent:* Tuesday, 26 May 2020 6:15 PM
*To:* dudaster/ele-custom-skin <ele-custom-skin@noreply.github.com>
*Cc:* andrewwpaction <andrew@wpaction.com.au>; Author <
author@noreply.github.com>
*Subject:* Re: [dudaster/ele-custom-skin] Alternating template is not
displaying a Text Field Correctly (#218)



As I can see you're using to Loop Templates:

   - one with the image on the right
   - and the other with the image on the left.

The one with the image on the left is not working. Chek the Text Editor
widget if it has the proper dynamic value set there.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/dudaster/ele-custom-skin/issues/218#issuecomment-633878594>,
or unsubscribe

**Comment by dudaster** [comment]
Please reply to github... I cannot see the attachements.

**Comment by dudaster** [comment]
Can you show me a screen shot when you edit the Loop Template with the image on the left?

**Comment by andrewwpaction** [comment]
Hi Her is all the Images I have been sending.

![Loop uses Post Content Field](https://user-images.githubusercontent.com/63601336/82886062-57c3b100-9f89-11ea-9069-0aec85828f56.jpg)
![Custom Posts with data edit](https://user-images.githubusercontent.com/63601336/82886070-598d7480-9f89-11ea-8faa-b552795d7450.jpg)
![Custom Posts with data](https://user-images.githubusercontent.com/63601336/82886072-5a260b00-9f89-11ea-8fb3-26bebf314fa6.jpg)
![Home Page loop edited in Elementor](https://user-images.githubusercontent.com/63601336/82886075-5abea180-9f89-11ea-93e7-503d635bd434.jpg)

**Comment by andrewwpaction** [comment]
Hi Just wondering how you are going with this. Any updates? Would you like access?

**Comment by dudaster** [comment]
Can you show me a screen shot when you edit the Loop Template with the image on the left?

**Comment by dudaster** [comment]
I still see the issue there can you provide me with the screenshot of the Loop Template that has the image on the left?
